metricbeat/module/elasticsearch/node_stats/_meta/fields.yml

```diff
@@ -1,158 +1,163 @@
-- name: node.stats
+- name: node
   type: group
   description: >
-    node_stats
+    Node
   release: beta
   fields:
-    - name: indices
+    - name: stats
```

Comment:
Not sure why you introduce the mapping here (was it for the other fields)? In the end both should work.

Reply:
Ugh, this change should've been reverted when I moved the node fields to the module level. Sorry about that. Fixing now...

Reply:
Reverted in b5912d88dc038d9adef72ff8f8ee13e782c37c0c.